spec/dummy/.env.example

```diff
@@ -0,0 +1,7 @@
+# This project uses environment variables for configuration
+# Copy .env.example and rename as .env in the project root
+# and add correct values. You can also use an environment specific file
+# e.g. .env.development
+
+DB_USER = "flood_risk"
```

Comment:
Yes - Using the PG_ prefix rather than DB makes sense. 

However, I don't think we should add environment variables that aren't used.